app/views/locker_applications/awaiting_assignment.html.erb

```diff
@@ -7,6 +7,10 @@
 
 <h1>Locker Applications</h1>
 
+<card size="full-width">
```

Comment:
Is the method `get` needed here?

Reply:
Possibly not necessary, retaining for clarity.